Add open problems page

kaiserd opened the pull request:
Add open problems page

---

(Below description a bit out of date)

This PR adds a research collaboration page.

My suggestion for funded positions is to have a JD for each of them and link the JD in the respective section of this research collaboration page.

Also, I'd not put in the "Research Proposal" section because it seems not very attractive for applicants. I just mentioned it for the sake of discussion during the offsite. We could make it more attractive by offering jointly writing the proposal. Still, I'd probably not list that.
Maybe we could write:
" We also offer jointly writing research proposals to relevant funding agencies in the listed research areas." 
My opinion: I'd not list this at all and keep that as an unlisted option in case we find an opportunity.

oskarth commented:
I suggest we take this PR and update it a bit to be in line with https://github.com/vacp2p/vac.dev/issues/82

Probably best to merge it after https://github.com/vacp2p/vac.dev/pull/79

kaiserd commented:
I reworked the open research problems page.

* removed the "funded positions" as we currently do not have any
* removed the anonymity analysis
  - Imo, only tasks we are not actively working, and that can be outsourced make sense here
  - during working on the anonymity analysis, such more contained topics might emerge and be added to the list

I'd suggest merging this soon and let other team members open their PR's to add more topics.

kaiserd commented:
NAT traversal might be an interesting topic to list here, too.
It does not have a high priority, but could be easily outsourced.

kaiserd commented on open-problems.md:
```diff
@@ -1,5 +1,18 @@
+We are seeking to collaborate with academia in the following topic areas.
+Feel free to join our [Discord](https://discord.gg/DXHGzzGX) for discussion.
+
+### DHT (distributed hash table) security / privacy / anonymity
```
I'd leave that just as a list of topics. This topic would be part of the discovery track, too.
No strong opinion here. Does anyone else have an opinion on this?

oskarth replied:
A list of topics makes sense to me, we can have research areas in () maybe.

oskarth replied:
Ideally we then have a list of up to 10 somewhat discrete problems. Basically this should be a live and updated version of https://vac.dev/vac-overview#problems-and-rough-priorities (and more detailed probably)

oskarth commented on open-problems.md:
```diff
@@ -1,5 +1,18 @@
+# Open Research Problems
 
-For now, please take a look at our open issues. For example in the [vacp2p/research repo](https://github.com/vacp2p/research/issues).
+We are seeking to collaborate with academia in the following topic areas.
```
Not only academia, also independent researchers - most people in the web3 space are more independent

The phrasing sounds a a bit too... dreary? Maybe say we have a lot of research areas and some specific open problems, and are excited to see contributions from people (affiliated or not)

oskarth commented:
Can we add more problems to this page? It feels a bit lacking. Perhaps going over with Sanaz (https://github.com/vacp2p/vac.dev/issues/81#issue-1321870804 per offsite) to make sure we capture a reasonable selection of problems that we've discussed previously, both in previous write-ups as well as things that have come up as we've been working on this research (MT optimization, node reputation, etc..)

staheri14 commented on open-problems.md:
```diff
@@ -1,5 +1,18 @@
 
-For now, please take a look at our open issues. For example in the [vacp2p/research repo](https://github.com/vacp2p/research/issues).
+We are seeking to collaborate with researchers in the following topic areas.
+Feel free to join our [Discord](https://discord.gg/DXHGzzGX) for discussion.
```
The Discord link does not seem to work.

staheri14 reviewed (approved):
Looks good as the start.

staheri14 commented:
> Can we add more problems to this page? It feels a bit lacking. Perhaps going over with Sanaz ([#81 (comment)](https://github.com/vacp2p/vac.dev/issues/81#issue-1321870804) per offsite) to make sure we capture a reasonable selection of problems that we've discussed previously, both in previous write-ups as well as things that have come up as we've been working on this research (MT optimization, node reputation, etc..)

I have a couple of topics in mind, let me compile their description and add to this page (if ok in a separate PR so that this one does not get blocked)
Some topics headlines: 
- Privacy-Preserving Incentivizations of waku protocols 
- Devising applications using Waku protocol stack e.g., a p2p social network, evoting system, etc.

oskarth reviewed (approved):
We can merge this as partial progress if this makes things simpler, but we need more work on this page before we can close issue (including on research front)

oskarth commented:
> Can we add more problems to this page? It feels a bit lacking. Perhaps going over with Sanaz (https://github.com/vacp2p/vac.dev/issues/81#issue-1321870804 per offsite) to make sure we capture a reasonable selection of problems that we've discussed previously, both in previous write-ups as well as things that have come up as we've been working on this research (MT optimization, node reputation, etc..)

> and let others open PRs adding their respective topics.

Can you @kaiserd and @staheri14 please do a sweep over e.g. previous write-ups and offsite discussion and add problems that we've already identified? Instead of it being "let others do it" :P

Doesn't have to be fancy, just that we capture various dimensions we've alluded to in various places.

oskarth commented:
>  Not trying to escape the work ;); Imo, it is more efficient if the person that came up with the topic, has more background knowledge, and is responsible for the respective track writes the respective description; they will know best what we need.

Right, I guess part of the point is that not everything is well-formulated, and what we want to capture on open problem space are especially things that might not be well-specified enough. I believe between you and Sanaz there's enough knowledge and understand that you should be able to cover most relevant areas, obv with other people reviewing + adding thoughts.
